components/Downloads/Release/VersionDropdown.tsx

```diff
@@ -21,19 +20,17 @@ const getDropDownStatus = (version: string, status: string) => {
 
 const VersionDropdown: FC = () => {
   const { releases, release, setVersion } = useContext(ReleaseContext);
-  const t = useTranslations();
 
   return (
     <Select
-      label={t('layouts.download.dropdown.version')}
```

Comment:
Also an reason for removing the label? Is it because it's inline? Should it also be inline on Mobile? How it interacts on mobile?

Reply:
No, there is no label on mobile
![Screenshot_20240329-013714.png](https://github.com/nodejs/nodejs.org/assets/86970828/2d39520e-2919-4a71-ad25-c83ef7a787c7)

Reply:
Right that's on the download page, but what about the other pages that use Router? I assume it's not affected, as the change is only on the RouterSelect?

How does the Select look when open on mobile?

Reply:
@ovflowd

No, this won't affect other pages, since this component is only used in 4 other components , 3 of which are located on `download page` and the other usage is in withRouterSelect component, which is HOC for page switcher, check the screenhsot
![image](https://github.com/nodejs/nodejs.org/assets/86970828/4737459b-0ef3-42ca-a2ac-b8b0bd8bf96c)

Reply:
I see. Well IMO on mobile screens we shouldn't use Inline Select as it can make navigation harder. Text becomes way smaller. Wdyt?

cc @bmuenzenmeyer